src/librbd/api/Group.cc

```diff
@@ -417,7 +418,7 @@ int Group<I>::remove(librados::IoCtx& io_ctx, const char *group_name)
   }
 
   r = Mirror<I>::group_disable(io_ctx, group_name, false);
-  if (r < 0 && r != -EOPNOTSUPP) {
+  if (r < 0 && r != -ENOENT && r != -EOPNOTSUPP) {
```

Comment:
This change seems out of place/redundant now.

Reply:
This is to follow the rest of the Group::remove() to skip ENOENT